src/docs/Components/Tag/index.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Tags'
+order: 6
+---
+
+> Tags are used to categorize, show filters or make selections. They display product properties.
+
+<Preview nude path="Tag" />
+
+## Four Tags types for different usages
+
+### Text tags
+
+Clickable tags are used to display relative keywords of an item or a content. They are used to create a cloud tag when a content live in a multiple categories.
+
+<Preview nude path="TagText" />
+
+### Link tags
+
+Link tags are a variation of the text tags, they have the same appearance but they are **used as a link**.
+
+<Preview nude path="TagLink" />
```

Comment:
The preview is strange, It need to group light tags together and dark tags together.